js/cointiger.js

```diff
@@ -405,6 +405,35 @@ module.exports = class cointiger extends huobipro {
         return this.parseTrades (response['data']['list'], market, since, limit);
     }
 
+    convertTimestamp (timestamp) {
+        timestamp = new Date (timestamp);
```

Comment:
We cannot transpile native JS datatypes, and we should use `ymd` instead. Can you rewrite this a bit and drop the Date and its methods from this file and replace them with the `ymd` method?